ext/pdo_odbc/tests/inc/odbc_pdo_test.inc

```diff
@@ -0,0 +1,66 @@
+<?php
+require_once dirname(__FILE__) . '/config.inc';
+require_once dirname(__FILE__) . '/../../../../ext/pdo/tests/pdo_test.inc';
+
+foreach ($env as $k => $v) {
+    define($k, $v);
+}
+
+class ODBCPDOTest extends PDOTest {
+
+    static function factory($classname = 'PDO') {
+        $dsn  = PDO_ODBC_TEST_DSN;
+        $user = PDO_ODBC_TEST_USER;
+        $pass = PDO_ODBC_TEST_PASS;
+        $attr = PDO_ODBC_TEST_ATTR;
+
+        $attr = is_string($attr) && strlen($attr) ? unserialize($attr) : null;
+
+        $db = new $classname($dsn, $user, $pass, $attr);
+        if (!$db) {
+            die("Could not create PDO object (DSN=$dsn, user=$user)\n");
+        }
+
+        $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_WARNING);
+        $db->setAttribute(PDO::ATTR_CASE, PDO::CASE_LOWER);
+        $db->setAttribute(PDO::ATTR_STRINGIFY_FETCHES, true);
+
+        return $db;
+    }
+
+    static function skip() {
+        if (substr(PHP_OS, 0, 3) == 'WIN' &&
+            false === getenv('PDOTEST_DSN') &&
+            false === getenv('PDO_ODBC_TEST_DSN') &&
+            !extension_loaded('com_dotnet')
+        ) {
+            die('skip - either PDOTEST_DSN or com_dotnet extension is needed to setup the connection');
+        }
+
+        try {
+            $db = self::factory();
+        } catch (PDOException $e) {
+            die('skip could not connect');
+        }
```

Comment:
I would like to eventually make parent skip like this, but the scope is too wide, so I won't change it yet in this PR.